translations/messages.fr.xlf

```diff
@@ -2287,6 +2295,82 @@
          Dans ce cas, le dernier joueur à avoir déplacé une pièce perd la partie.
          Ici, votre adversaire a déplacé sa pièce verte en dernier, vous êtes donc vainqueur !</target>
       </trans-unit>
+      <trans-unit id="2020802782007615776" datatype="html">
+        <source>Capture must be double diagonal steps</source>
+        <target>Les captures doivent être des sauts de deux pas en diagonal</target>
+      </trans-unit>
+      <trans-unit id="3722504673049657082" datatype="html">
+        <source>Move must be single diagonal steps</source>
+        <target>Les déplacements doivent être des simple pas en diagonal</target>
+      </trans-unit>
+      <trans-unit id="8722600729890521244" datatype="html">
+        <source>You cannot jump over the same coord several time</source>
+        <target>Vous ne pouvez pas sauter au dessus d'une même case plusieurs fois</target>
+      </trans-unit>
+      <trans-unit id="8833682869379041729" datatype="html">
+        <source>Cannot go backward with normal piece</source>
+        <target>Vous ne pouvez pas aller en arrière avec une pièce normale</target>
+      </trans-unit>
+      <trans-unit id="8649496248690252329" datatype="html">
+        <source>Cannot capture empty space (nor jump over it)</source>
+        <target>Vous ne pouvez pas capturer une case vide (ni sauter par dessus)</target>
+      </trans-unit>
+      <trans-unit id="8144038524408187730" datatype="html">
+        <source>Capture is mandatory</source>
+        <target>La capture est obligatoire</target>
+      </trans-unit>
+      <trans-unit id="4753584102674404468" datatype="html">
+        <source>You must finish this capture</source>
+        <target>Vous devez finir cette capture</target>
+      </trans-unit>
+      <trans-unit id="8274439231274366704" datatype="html">
+        <source>Lasca: origins</source>
+        <target>Lasca: les origines</target>
+      </trans-unit>
+      <trans-unit id="9130733749994426297" datatype="html">
+        <source>Lasca is a game based on draughts created in 1911 by Emanuel Lasker, chess world champion. It&apos;s played on a 7x7 board, each players has 11 pieces.</source>
+        <target>Lasca est un jeu basé sur le jeu de dames, crée en 1911 par Emanuel Lasker, champion du monde d'échecs. Lasca se joue sur un plateau de 7x7 cases, chaque joueur a 11 pièces.</target>
+      </trans-unit>
+      <trans-unit id="3198994967671406143" datatype="html">
+        <source>The goal of Lasca is, like for draughts, to put the opponent out of possible move, either by capturing all his pieces, either by blocking them.</source>
```

Comment:
to render the opponent unable to move

Reply:
not done

Reply:
euh, remet le commentaire sur un extrait de code pas outdaté alors, je trouve pas ça dans mon code :D